OpenRobertaServer/staticResources/js/app/roberta/controller/import.controller.js

```diff
@@ -81,7 +81,18 @@ define([ 'exports', 'comm', 'message', 'log', 'util', 'guiState.controller', 'pr
                     MSG.displayInformation(result, "", Blockly.Msg.ORA_PROGRAM_IMPORT_ERROR, name);
                 }
             } else {
-                MSG.displayInformation(result, "", result.message, name);
+                //Find a way to get specific robot type
```

Comment:
You don't need the `result.parameters` array here if you just use `result.robotType` and pass that instead of name in line 95. That should work around the added "program" suffix from earlier.